tests/network/vmi_istio.go

```diff
@@ -179,7 +179,7 @@ var istioTests = func(vmType VmType) {
 				libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(virtClient, migration)
 			})
 			It("All containers should complete in source virt-launcher pod after migration", func() {
-				const containerCompletionWaitTime = 60
+				const containerCompletionWaitTime = 240
```

Comment:
you might want to use  `disableSerialConsoleLog` on istio tests
it removes one container that sometimes stall the load 
and this test mentions it waits for containers to complete

./cluster/kubectl.sh -n kubevirt patch kubevirt kubevirt --type=merge --patch '{"spec":{"configuration":{"virtualMachineOptions":{"disableSerialConsoleLog":{}}}}}'

There is an issue about it https://github.com/kubevirt/kubevirt/issues/12771
that closed due to inactivity, not sure if it was solved or not
but it can make virt-launcher lighter so it is good to disable it anyhow